tests/fail/both_borrows/aliasing_mut1.tree.stderr

```diff
@@ -16,7 +16,8 @@ help: the accessed tag <TAG> later transitioned to Frozen due to a foreign read
    |
 LL | pub fn safe(x: &mut i32, y: &mut i32) {
    |                          ^
-   = help: this corresponds to a loss of write permissions
+   = help: this transition corresponds to a loss of write permissions
+   = help: the read access that caused this transition was inserted implicitly due to a reborrow
```

Comment:
I think what I'd like to see here is a change already further up in this message:
```
help: the accessed tag <TAG> later transitioned to Frozen due to a foreign reborrow at offsets [0x0..0x4]
  --> $DIR/aliasing_mut1.rs:LL:CC
   |
LL | pub fn safe(x: &mut i32, y: &mut i32) {
   |                          ^
   = help: this transition corresponds to a loss of write permissions
```
Maybe with another line saying
```
   = help: reborrows act like read access for the purpose of the aliasing rules
```
though I am not sure if that is needed (we don't have it for SB)

Reply:
Current phrasing gives both: "the accessed tag <TAG> later transitioned to Frozen due to a reborrow (acting as a foreign read access) at offsets [0x0..0x4]"